{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/sections/assets/css/section.css

```diff
@@ -0,0 +1,126 @@
+/*
+|--------------------------------------------------------------------------
+| Section
+|--------------------------------------------------------------------------
+| @namespace: sec-
+|
+*/
+.sec-Section-transparent {
+  /* Transparent sections (background the same colour as the body) use
+  margins to space them to avoid the illusion of excessive padding -
+  margins collapse on adjacent sections.
+
+  If you change these spacings, make sure you adjust the spacing for
+  sections with a coloured background below. */
+  margin-top: 1vr;
+  margin-bottom: @margin-top;
+
+  @media (--sm) {
+    margin-top: 1.5vr;
+    margin-bottom: @margin-top;
+  }
+
+  @media (--md) {
+    margin-top: 2vr;
+    margin-bottom: @margin-top;
+  }
+
+  @media (--lg) {
+    margin-top: 3vr;
+    margin-bottom: @margin-top;
+  }
+
+  @media (--xlg) {
+    margin-top: 4vr;
+    margin-bottom: @margin-top;
+  }
+}
+
+.sec-Section-coloured {
+  padding-top: 1vr;
+  padding-bottom: @padding-top;
+
+  @media (--sm) {
+    padding-top: 1.5vr;
+    padding-bottom: @padding-top;
+  }
+
+  @media (--md) {
+    padding-top: 2vr;
+    padding-bottom: @padding-top;
+  }
+
+  @media (--lg) {
+    padding-top: 3vr;
+    padding-bottom: @padding-top;
+  }
+
+  @media (--xlg) {
+    padding-top: 4vr;
+    padding-bottom: @padding-top;
+  }
+}
+
```

Comment:
What are the benefits of using margins over padding? Couldn't we use padding to space all sections then we only need modifier classes  to change background-colour and colour.

Reply:
The reasoning behind this is that if you have two transparent (usually white-backgrounded) sections next to each other, the spacing between them will be enormous; this is not normally a problem for colour-backgrounded sections. Margins, on the other hand, collapse between adjacent items - the largest margin will take precedence.